src/chain_spec.rs

```diff
@@ -734,6 +734,8 @@ fn centrifuge_genesis(
 				.to_vec(),
 		},
 		balances: centrifuge_runtime::BalancesConfig { balances },
+		orml_asset_registry: Default::default(),
```

Comment:
Maybe a different name would be more meaningful for `Default::default()`?

Reply:
Note that `Default` is a trait, so what this is stating is that the genesis value for the asset-registry storage is `default`, i.e, empty/nothing. This approach is used to a handful of other genesis entries 👍